ffyaml/ffyaml.go

```diff
@@ -68,6 +71,8 @@ func valToStr(val interface{}) (string, error) {
 		return strconv.FormatInt(v, 10), nil
 	case float64:
 		return strconv.FormatFloat(v, 'g', -1, 64), nil
+	case nil:
+		return "", nil
```

Comment:
So now this is the only substantive change in the PR?

Reply:
That is correct. Everything else is just Unit test or related to try to self-document what the behavior is in the case of Blank YAM entries to FlagSet entries of various types. 

I am open to amending the unit test & fixture code. But, my whole goal with that stuff was to be able show to other lib users of ff, what the expected YAML parser behavior is with this particular test case, i.e, classic T-D-D, unit test/fixture code as clear, explicit documentation of one particular behavioral edge-case...that I happened to run into.